viewer/views/stats.jade

```diff
@@ -767,8 +771,10 @@ block content
         } );
       }
 
-      $('#statsGraphSelect').change(function() {
-        makeStatsGraph($('#statsGraphSelect').val());
+      $('#statsGraphSelect,#statsTimeSelect').change(function() {
+        var interval = + $('#statsTimeSelect').val();
```

Comment:
I'd use `parseInt` here instead of `+`. Only because (1) more people understand parseInt, and (2) it's a teensy tiny bit faster in most browsers. Also, `+` is technically closer to `parseFloat` since it also accepts decimals. But really, for this case, it doesn't matter because we only have a few known values.